jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -889,52 +889,6 @@ default Throwable getFailure()
          */
         boolean isLast();
 
-        /**
-         * <p>Copies the bytes from this Chunk to the given byte array.</p>
-         *
-         * @param bytes the byte array to copy the bytes into
-         * @param offset the offset within the byte array
-         * @param length the maximum number of bytes to copy
-         * @return the number of bytes actually copied
-         */
-        default int get(byte[] bytes, int offset, int length)
-        {
-            ByteBuffer b = getByteBuffer();
-            if (b == null || !b.hasRemaining())
-                return 0;
-            length = Math.min(length, b.remaining());
-            b.get(bytes, offset, length);
-            return length;
-        }
-
-        /**
-         * <p>Skips, advancing the ByteBuffer position, the given number of bytes.</p>
-         *
-         * @param length the maximum number of bytes to skip
-         * @return the number of bytes actually skipped
-         */
-        default int skip(int length)
-        {
-            if (length == 0)
-                return 0;
-            ByteBuffer byteBuffer = getByteBuffer();
-            length = Math.min(byteBuffer.remaining(), length);
-            byteBuffer.position(byteBuffer.position() + length);
-            return length;
-        }
-
-        /**
-         * @return an immutable version of this Chunk
-         */
-        default Chunk asReadOnly()
```

Comment:
Unfortunately, I think this method can only be deprecated for removal, as it is currently part of the API.

Reply:
Ah good point

Reply:
... unless we make this PR a 12.1 thing.... in which case we can deprecate in 12.0.x and remove here.